sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -1025,3 +1025,5 @@ case class ShowCreateTableCommand(table: TableIdentifier) extends RunnableComman
     builder.toString()
   }
 }
+
+trait InsertTableCommand extends RunnableCommand
```

Comment:
This might be too broad for this purpose. It could be misunderstood later. Should we add a comment for it?